packages/merge-styles/src/Stylesheet.ts

```diff
@@ -15,6 +17,21 @@ export const InjectionMode = {
    * Appends rules using appendChild.
    */
   appendChild: 2 as 2,
+
+  /**
+   * Inserts rules into constructable stylesheets.
+   */
+  constructableStylesheet: 3 as 3,
+
+  /**
+   * Same as `insertNode` and `constructableStylesheet`
+   */
+  insertNodeAndConstructableStylesheet: 4 as 4,
+
+  /**
+   * Same as `appendChild` and `constructableStylesheet`
+   */
+  appedChildAndConstructableStylesheet: 5 as 5,
 };
```

Comment:
I'm game but does that change the type in a breaking way (I don't know enough to say)?

Reply:
Looks like it might be because it adds readonly?
|original|requested change|
|---|---|
|![image](https://github.com/microsoft/fluentui/assets/5953191/1c96e58a-c620-4e5a-8410-1f9f4bea6ba9)|![image](https://github.com/microsoft/fluentui/assets/5953191/69066bd4-2d2c-4cc2-8ceb-1c317f478f60)|

Reply:
I think I'll leave this as-is for now since the priniciples for this feature include it being entirely opt-in and it having minimal change to the existing implementation. I agree that this suggestion is better but it seems like it subtly changes the types and I think we should avoid getting into those types of changes here.